.github/workflows/ci.yml

```diff
@@ -141,10 +141,9 @@ jobs:
           - plugin
           - standalone
         engine:
-          - 24.0.9
           - 25.0.5
           - 26.1.4
-          - 27.4.0-rc.4
+          - 27.4.1
```

Comment:
I just had a check if the script would be fine with that, but it looks like it is; it's possible to specify just `27.4` or even just `27`, and it will pick the latest `27.4.x` or `27.x.x`. That may be ok to consider to not having to update for each minor or patch release.


I'll also check with @vvoland and @crazy-max to see if we can swap this out for our new `docker-setup-action` (https://github.com/docker/setup-docker-action), but that can definitely wait for separate for sure!